patterns/1-initial/walk-the-innersource-talk.md

```diff
@@ -0,0 +1,150 @@
+## Title
+
+Walk the InnerSource talk
+
+## Patlet
+
+Teams across the organization are encouraged to adopt InnerSource principles such as working openly, sharing code, and
+collaborating transparently. But, if the team behind the InnerSource initiative doesn’t follow these practices
+themselves, it undermines credibility and adoption. Therefore, this team should lead by example: documenting their
+decisions as code, working in the open, and treating their work as an InnerSource project to build trust and show others
+how it’s done.
+
+## Problem
+
+The team behind the InnerSource initiative promotes transparency, collaboration, and best practices across the
+organization. However, if the team itself does not adhere to these principles, others may perceive InnerSource as mere
+rhetoric rather than a transformative practice. Without leading by example, adoption and trust in InnerSource
+initiatives may suffer.
+
+## Story (optional)
+
+At Siemens, when the InnerSource journey began, many of the early contributors were deeply inspired by the Open Source
+culture, several were also active participants in community projects. For them, setting up an InnerSource platform
+wasn’t a stretch; it was a natural continuation of how they already worked: asynchronously across locations, sharing
+ideas through issues, and documenting decisions transparently.
+
+This Open Source-inspired mindset gave the InnerSource platform a distinctly different character - transparent,
+collaborative, and developer-friendly - compared to the closed, proprietary environments many developers were used to.
+Unsurprisingly, it quickly attracted experts and engineers seeking more open, cross-team collaboration and meaningful
+technical exchange.
+
+## Context
+
+In many large organizations, IT services rely on what we might call a “TicketOps” model: users submit requests into a
+system, but the people and decisions behind the process remain invisible. This lack of transparency often breeds
+frustration and mistrust, making true collaboration across departments difficult.
+
+In contrast, developer-centric initiatives like InnerSource thrive on openness and shared ownership. Instead of hiding
+behind tickets, teams work in open repositories, discuss decisions transparently, and empower others to contribute. This
+shift fosters trust, reduces the frustration of not knowing what happens behind the scenes, accelerates problem-solving,
+and turns support into collaboration.
+
+## Forces
+
+### What makes the problem difficult?
+
+- In large, traditionally structured organizations, teams - especially platform, architecture, or governance groups -
+are often used to working behind closed doors. Decision-making happens in meetings, documentation is internal or
+scattered, and ownership is unclear. Shifting to transparent, open collaboration requires both cultural change and a
+reevaluation of existing workflows.
```

Comment:
Maybe we can mention here somehow that working this way sets a quite high bar, where people might not feel "good enough" to contribute.

Reply:
Maybe like this:

```suggestion
- In large, traditionally structured organizations, teams - especially platform, architecture, or governance groups -
are often used to working behind closed doors. Decision-making happens in meetings, documentation is internal or
scattered, and ownership is unclear. Shifting to transparent, open collaboration requires both cultural change and a
reevaluation of existing workflows.
- Working in the open can feel scary - when everything is public, people might hold back from contributing because they worry their work isn't perfect enough. This fear of not being "good enough" can stop many valuable ideas from being shared.
```

Reply:
Nice! I like the idea of addressing fears...